action.yml

```diff
@@ -11,6 +11,10 @@ inputs:
     description: "An optional id to use for this message."
     default: "add-pr-comment"
     required: false
+  auto-refresh-message-position:
+    description: "If a message with the same id, this option allow to refresh the position of the message to be the last one posted."
+    default: "true"
```

Comment:
Would prefer this be `default: "false"` unless we want to make this a major version bump. Maybe once the feature is out we can do a small poll and change the default for v3 if folks prefer it.

Reply:
You're right.